setup.cfg

```diff
@@ -26,8 +26,20 @@ artifact_indexes=
     https://pypi.anaconda.org/scikit-learn-wheels-staging/simple/scikit-learn/
 
 [flake8]
+# max line length for black
+max-line-length = 88
+target-version = ['py37']
 # Default flake8 3.5 ignored flags
-ignore=E121,E123,E126,E226,E24,E704,W503,W504
+ignore=E121,E123,E126,E226,E24,E704,W503,W504,E203,E731,E741
```

Comment:
```suggestion
ignore=
    E24,   # What does this mean?
    E121,  # continuation line under-indented
    E123,  # closing bracket does not match indentation
    E126,  # continuation line over-indented for hanging indent
    E203,  # space before : (needed for how black formats slicing)
    E226,  # missing whitespace around arithmetic operator
    E704,  # multiple statements on one line (def)
    E731,  # do not assign a lambda expression, use a def
    E741,  # do not use variables named ‘l’, ‘O’, or ‘I’
    W503,  # line break before binary operator
    W504   # line break after binary operator
```
Pandas does that, too. See https://github.com/pandas-dev/pandas/blob/master/setup.cfg

What's the meaning of E24? I couldn't find its meaning.

Reply:
Hmm, I also couldn't find what E24 means but it's part of the [error codes ignored by default]( https://flake8.pycqa.org/en/latest/manpage.html) so we have no reason to enable it.

Reply:
I'll merge your suggestion and PR. I don't think it's worth blocking it to investigate what E24 does.

Reply:
> What's the meaning of E24? I couldn't find its meaning.

I think that `E24` refers to the E23x family of errors.

E241: multiple spaces after separator
E242: tab after separator